fmriprep/workflows/bold/confounds.py

```diff
@@ -412,10 +412,22 @@ def init_bold_confs_wf(
         name='crowncc_metadata_fmt',
     )
 
+    # Combine all confounds metadata
     mrg_conf_metadata = pe.Node(
-        niu.Merge(3), name='merge_confound_metadata', run_without_submitting=True
+        niu.Merge(4), name='merge_confound_metadata', run_without_submitting=True
     )
+    # Tissue mean time series
     mrg_conf_metadata.inputs.in3 = {label: {'Method': 'Mean'} for label in signals_class_labels}
+    # Movement parameters
+    mrg_conf_metadata.inputs.in4 = {
+        'trans_x': {'Description': 'Translation Along X Axis', 'Units': 'mm'},
+        'trans_y': {'Description': 'Translation Along Y Axis', 'Units': 'mm'},
+        'trans_z': {'Description': 'Translation Along Z Axis', 'Units': 'mm'},
+        'rot_x': {'Description': 'Rotation Along X Axis', 'Units': 'rad'},
+        'rot_y': {'Description': 'Rotation Along Y Axis', 'Units': 'rad'},
+        'rot_z': {'Description': 'Rotation Along Z Axis', 'Units': 'rad'},
+        'framewise_displacement': {'Units': 'mm / TR'},
```

Comment:
It strikes me as odd to call it a velocity instead of a distance. Unless there's precedent in the literature you're following?

```suggestion
        'framewise_displacement': {'Units': 'mm'},
```

Reply:
Isn't it actually a measure of instantaneous velocity though? One could scale it based on seconds instead of TRs (i.e., as something like "secondwise_displacement"), and the units would be different, even if the distance unit (mm) was the same.

Reply:
I think to be precise, it's a change in displacement with respect to the index: $\Delta x_i / \Delta i$. Indices are unitless so the quantity is in the original units. Granted, for most BOLD sequences, $t_i = t_0 + T_R*i$, but it is not true in all cases, and it plays no role in the calculation.

Apart from the pedantic point that I'm not 100% confident in, I don't think I've ever seen it written as `mm / TR`, so this would be something we're introducing; and BIDS units are supposed to be SI, which makes this awkward if anybody ever tries to validate it.

Reply:
Ah, I see. I'll change it in that case.

Regarding units though, TR _is_ a special value for SamplingFrequency in BEP012, so it seems like we should have it as a unit in BIDS at some point. Or maybe support SamplingFrequency as a unit.